Changelog.md

```diff
@@ -1,5 +1,117 @@
 # Changelog
 
+### 0.22.0
+
+**Image:** `quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.22.0`
+
+_New Features:_
+
+- NGINX 1.15.8
+- New balancer implementation: consistent hash subset
+- Adds support for HTTP2 Push Preload annotation
+- Allow to disable NGINX prometheus metrics
+- Add --disable-catch-all option to disable catch-all server
```

Comment:
What about:

New `--disable-catch-all` flag to ignore catch-all ingresses

Reply:
changed